data/showcase.yml

```diff
@@ -34,6 +34,18 @@
   features:
     - A more full-featured Ember web app in action
     - Usage of <a href="https://ember-fastboot.com/">Ember FastBoot</a>.
+- name: Deprecations
+  image:
+    src: showcase-deprecations.png
+    alt: Deprecations application
+  repository: https://github.com/ember-learn/deprecation-app
+  demo: https://emberjs.com/deprecations
+  description: |
+    This application was built to showcase Ember deprecations at <a href="http://emberjs.com/deprecations/">deprecations app</a>.  You'll be able to learn and identify deprecations for each major release such as:
+  features:
```

Comment:
I think the description could mention some of the tech used as features instead of the content, since we talk so much about deprecations elsewhere on this page. For example:
- Pre-renders content for faster loading, using prember and Fastboot
- Parses the source material from markdown files using ember-prism

The addons used should be links.